Reconstruction/RecCalorimeter/src/components/ConstNoiseTool.cpp

```diff
@@ -0,0 +1,93 @@
+#include "ConstNoiseTool.h"
+
+// FCCSW
+#include "DetCommon/DetUtils.h"
+#include "DetInterface/IGeoSvc.h"
+
+// DD4hep
+#include "DD4hep/Detector.h"
+
+// Root
+#include "TFile.h"
+#include "TH1F.h"
+#include "TMath.h"
+
+DECLARE_TOOL_FACTORY(ConstNoiseTool)
+
+ConstNoiseTool::ConstNoiseTool(const std::string& type, const std::string& name, const IInterface* parent)
+    : GaudiTool(type, name, parent) {
+  declareInterface<INoiseConstTool>(this);
+}
+
+StatusCode ConstNoiseTool::initialize() {
+  /// Noise estimates from Atlas [GeV]
```

Comment:
Hi Anna, this could be used also for other calo systems, instead of hard coding the "noise" level, I could give the tool two arguments which set the values for the EM and HAD systems.

Reply:
Ok, but then remove those lines (including the comment) and use the properties instead. You can put those as defaults and comment on them.